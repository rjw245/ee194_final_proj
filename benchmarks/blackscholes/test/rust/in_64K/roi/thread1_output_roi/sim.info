{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread1_output_roi',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--roi',
             '--',
             '../../../../rust/bs',
             '1',
             '../../../../inputs/in_64K.txt'],
 'git_revision': '',
 'host': 'dell25',
 'pin_version': 71313,
 'rusage': (31805.279862,
            17.7593,
            117460,
            0,
            0,
            0,
            274243,
            0,
            0,
            0,
            12056,
            0,
            0,
            0,
            269056,
            5519046),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/rust/in_64K/roi/thread1_output_roi --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --general/magic=true -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../../rust/bs 1 ../../../../inputs/in_64K.txt'],
 't_elapsed': 51434.1651802063,
 't_start': 1461850778.113381,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 3429466112L}
