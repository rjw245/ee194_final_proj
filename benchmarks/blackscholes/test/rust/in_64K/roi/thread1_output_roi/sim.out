                                     | Core 0       | Core 1      | Core 2      | Core 3      | Core 4      | Core 5      | Core 6      | Core 7     
  Instructions                       |  10752572308 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  Cycles                             |   9670570366 |  9670570366 |  9670570366 |  9670570366 |  9670570366 |  9670570366 |  9670570366 |  9670570366
  IPC                                |         1.11 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  Time (ns)                          |   3635552770 |  3635552770 |  3635552770 |  3635552770 |  3635552770 |  3635552770 |  3635552770 |  3635552770
  Idle time (ns)                     |        20110 |  3635552770 |  3635552770 |  3635552770 |  3635552770 |  3635552770 |  3635552770 |  3635552770
  Idle time (%)                      |         0.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0%
Branch predictor stats               |              |             |             |             |             |             |             |            
  num correct                        |    391050358 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num incorrect                      |     13124594 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  misprediction rate                 |        3.25% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
  mpki                               |         1.22 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
TLB Summary                          |              |             |             |             |             |             |             |            
  I-TLB                              |              |             |             |             |             |             |             |            
    num accesses                     |   1660404609 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num misses                       |            8 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |        0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |         0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                              |              |             |             |             |             |             |             |            
    num accesses                     |   6150797340 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num misses                       |        38533 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |        0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |         0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  L2 TLB                             |              |             |             |             |             |             |             |            
    num accesses                     |        38541 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num misses                       |        13146 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       34.11% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |         0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
Cache Summary                        |              |             |             |             |             |             |             |            
  Cache L1-I                         |              |             |             |             |             |             |             |            
    num cache accesses               |   1660404609 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |        65007 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |        0.00% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |         0.01 |         inf |         inf |         inf |         inf |         inf |         inf |         inf
  Cache L1-D                         |              |             |             |             |             |             |             |            
    num cache accesses               |   6150797340 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |      2873795 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |        0.05% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |         0.27 |         inf |         inf |         inf |         inf |         inf |         inf |         inf
  Cache L2                           |              |             |             |             |             |             |             |            
    num cache accesses               |      2938802 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |      2665006 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       90.68% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |         0.25 |         inf |         inf |         inf |         inf |         inf |         inf |         inf
  Cache L3                           |              |             |             |             |             |             |             |            
    num cache accesses               |      2665006 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |          362 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |        0.01% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |         0.00 |         inf |         inf |         inf |         inf |         inf |         inf |         inf
DRAM summary                         |              |             |             |             |             |             |             |            
  num dram accesses                  |          181 |           0 |           0 |           0 |         181 |           0 |           0 |           0
  average dram access latency (ns)   |        59.00 |         inf |         inf |         inf |       56.54 |         inf |         inf |         inf
  average dram queueing delay        |         5.57 |        0.00 |        0.00 |        0.00 |        3.11 |        0.00 |        0.00 |        0.00
  average dram bandwidth utilization |        0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
Coherency Traffic                    |              |             |             |             |             |             |             |            
  num loads from dram                |          272 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from dram cache          |            0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from remote cache        |            0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
